back/internal/postsvc/internal/postdb/qrypost.go

```diff
@@ -72,24 +110,27 @@ func (s *PostDB) upsertPost(ctx cx, sid string, x *pb.AddPostReq, y *pb.PostResp
 // TODO: make it such that if given a list of multiple keywords, we can search the
 // title and body for those keywords.
 func (s *PostDB) findPosts(ctx cx, x *pb.FndPostsReq, y *pb.PostsResp) error {
-	selStmt, err := s.db.Prepare("SELECT post_id, type_id, slug, title, body FROM post WHERE title LIKE '%?%' OR body like '%?%'")
+	selStmt, err := s.db.Prepare("SELECT post_id, type_id, slug, title, body FROM post WHERE title LIKE ? OR body LIKE ?")
```

Comment:
It would be nice to have a defined approach to this endpoint. There should be a primitive endpoint that provides general, direct, and simple access with a separate and more complex endpoint that provides specific, indirect, and complex access (fuzzy search). By splitting at those concerns, we can guard usage of each endpoint differently and with less entanglements than if they are a unified feature. Again, we'll talk.